addons/zeus/stringtable.xml

```diff
@@ -1037,41 +1049,51 @@
         </Key>
         <Key ID="STR_ACE_Zeus_ModuleLoadIntoCargo_DisplayName">
             <English>Load into Cargo</English>
+            <German>In Frachtraum laden</German>
         </Key>
         <Key ID="STR_ACE_Zeus_moduleToggleNVG_DisplayName">
             <English>Toggle NVGs</English>
+            <German>Ändere Nachtsichtgeräte</German>
             <French>Basculer JVN</French>
         </Key>
         <Key ID="STR_ACE_Zeus_moduleToggleNVG_ToggleNvgTitle">
             <English>NVG equipment</English>
+            <German>Nachtsichtgeräte</German>
             <French>Equipment de vision nocturne</French>
         </Key>
         <Key ID="STR_ACE_Zeus_moduleToggleNVG_ToggleNvgTitleTooltip">
             <English>Add or remove NVGs from units</English>
+            <German>Hinzufügen/Entfernen von Nachtsichtgeräten</German>
             <French>Ajouter ou retirer JVN des unités</French>
         </Key>
         <Key ID="STR_ACE_Zeus_moduleToggleNVG_ToggleNvgSide">
             <English>Toggle NVG target</English>
+            <German>Ändere Nachtsichtgeräte Ziel</German>
             <French>Cible du basculement</French>
         </Key>
         <Key ID="STR_ACE_Zeus_moduleToggleNVG_SelectedGroup">
             <English>Selected group</English>
+            <German>Wähle eine Gruppe</German>
             <French>Groupe sélectionné</French>
         </Key>
         <Key ID="STR_ACE_Zeus_moduleToggleFlashlight_DisplayName">
             <English>Toggle flashlights</English>
+            <German>Ändere Taschenlampen</German>
             <French>Basculer lampes torches</French>
         </Key>
         <Key ID="STR_ACE_Zeus_moduleToggleFlashlight_ToggleFlashlightSide">
             <English>Toggle flashlight target</English>
+            <German>Ändere Nachtsichtgeräte Ziel</German>
```

Comment:
passt nicht

@alganthe That's probably the same as above just with flash light.
Btw. the french translation below is the same as with the NVGs.